composer/utils/checkpoint.py

```diff
@@ -289,6 +290,7 @@ def read_data(self, plan: LoadPlan, planner: LoadPlanner):
         if self.device_mesh is not None and self.device_mesh.ndim == 2:
             # Broadcast file to all replicas
             replicate_process_group = self.device_mesh.get_group(0)
+            hsdp_shard_process_group = self.device_mesh.get_group(1)
```

Comment:
```suggestion
            shard_process_group = self.device_mesh.get_group(1)
```
replicate group does not say hsdp, to be consistent